src/Common/DirectMessaging/ErrorConstants.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * ErrorConstants holds a number of string constants and error codes
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <snielson@discoverandchange.com>
+ * @copyright Copyright (c) 2022 Stephen Nielson <snielson@discoverandchange.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Common\DirectMessaging;
+
+class ErrorConstants
+{
+    // Message that is sent back to the user (translated) if messaging is unavailable
+    const MESSAGING_DISABLED = "Direct messaging is currently unavailable.";
+
+    // used in the message sent back to the user when something fails
+    const ERROR_CODE_ABBREVIATION = "EC";
+
+    // translated message sent back to the user when a Direct Address is invalid or message delivery not allowed
+    // Details of the reason is attached to this message (which is not translated).
+    const RECIPIENT_NOT_ALLOWED = "Delivery is not allowed to the specified Direct Address: ";
```

Comment:
this constant will need to have the ending space removed to ensure translated in current engine (would place the space in the place where using this constant)